Rules/AvoidGlobalFunctions.cs

```diff
@@ -0,0 +1,205 @@
+﻿using Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Globalization;
+using System.Management.Automation.Language;
+
+namespace Microsoft.Windows.PowerShell.ScriptAnalyzer.BuiltinRules
+{
+    [Export(typeof(IScriptRule))]
```

Comment:
This needs to the directives as well. Please have a look at [AvoidGlobalVars](https://github.com/PowerShell/PSScriptAnalyzer/blob/development/Rules/AvoidGlobalVars.cs)